beacon-chain/slasher/detect_attestations.go

```diff
@@ -57,11 +64,14 @@ func (s *Service) detectSlashableAttestations(
 		return err
 	}
 
-	totalSlashings := append(slashings, moreSlashings...)
-	if len(totalSlashings) > 0 {
-		log.WithField("numSlashings", len(totalSlashings)).Info("Slashable offenses found")
+	// Consolidate all slashings into a slice.
+	slashings = append(slashings, doubleVoteSlashings...)
+	slashings = append(slashings, surroundingSlashings...)
+	slashings = append(slashings, surroundedSlashings...)
+	if len(slashings) > 0 {
+		log.WithField("numSlashings", len(slashings)).Info("Slashable offenses found")
```

Comment:
I think we can log more info here. What kind of offense and which validator index committed the offense?

Reply:
Nvm, it looks like we do log the more details after